public/static/index.html

```diff
@@ -24,6 +24,20 @@
 
     <script src="https://identity.netlify.com/v1/netlify-identity-widget.js"></script>
     <script src="https://kit.fontawesome.com/491fca5fd9.js" crossorigin="anonymous"></script>
+
+    <!-- Global site tag (gtag.js) - Google Analytics -->
+    <script async src="https://www.googletagmanager.com/gtag/js?id=G-13GYPPVD4Y"></script>
+    <script>
+      window.dataLayer = window.dataLayer || [];
+      function gtag(){dataLayer.push(arguments);}
+      gtag('js', new Date());
+
+      gtag('config', 'G-13GYPPVD4Y', {
+        'allow_google_signals': false,
+        'allow_ad_personalization_signals': false,
+        'debug_mode': '<%= htmlWebpackPlugin.options.isDev %>'
+      });
+   </script>
```

Comment:
can we inject the environmental variables for `GA_MEASUREMENT_ID`?

also can define the gtag config there as well

Reply:
ok